docs/components/Combobox/Combobox.stories.mdx

```diff
@@ -28,19 +28,37 @@ Combobox is composed of three main parts:
 #### Trigger
 
 The trigger is the user-facing interface to open and close the Combobox and
-initiate option selection. The choice between them depends on the design and
-user experience requirements.
-
-There are two options for trigger elements of the Combobox,
-[TriggerButton](../?path=/story/components-selections-combobox-web--trigger-button)
+initiate option selection. When implementing Combobox, you may rely on the
+default trigger or, you may pass in a custom activator.
+
+The default trigger is a
+[Chip](../?path=/story/components-selections-chip-web--subtle) containing a
+label that reads "Select" and an `add` Icon suffix. The label can be customized
+by adding the optional `label` prop to better reflect the type of data being
+selected. Examples of using a default trigger can be found in the
+[single-select](../?path=/code/components-selections-combobox-web--single-select)
 and
-[TriggerChip](../?path=/story/components-selections-combobox-web--trigger-chip).
-You must have one trigger component.
+[multi-select](../?path=/code/components-selections-combobox-web--multi-select)
+stories.
+
+Notice that omitting a trigger subcomponent will result in the default trigger
```

Comment:
nitpick: Trigger was our old wording, now it's the Activator which might be a better word to use here

though I do get that we're under a heading called `Trigger` and discussing that topic, so maybe it's fine?

Reply:
I always disliked the word trigger anyway, so everything is changed to say activator. Technically the `ComboboxTrigger` does still exist as the default but that word won't surface to the user.